### PR TITLE
Simple patch for using `GetModule<T>();` instead of array selectors.

### PR DESCRIPTION
**Pull request by ghost**
# Pull Request #2188 Info/Notes

## Changes
- Switched out the array system used for the stat modules with `GetModule<T>();`.

## More Notes
- None

**End of PR**

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -838,7 +838,7 @@ public float Health
             get => healthStat.CurValue;
             set
             {
-                healthStat.CurValue = value;
+                ReferenceHub.playerStats.GetModule<HealthStat>().CurValue = value;
```
Comment (louis1706) [inline Exiled.API/Features/Player.cs:841]: This is actually wrong

Comment (ALEXWARELLC) [reply Exiled.API/Features/Player.cs:841]: Whoops. I'll look over it.

Comment (ALEXWARELLC) [reply Exiled.API/Features/Player.cs:841]: > This is actually wrong

I'm unable to look into this until Visual Studio stops being so dumb. But feel free to correct me.

**Comment by ALEXWARELLC** [comment]
Man, the [Halloween Branch](https://github.com/Exiled-Team/EXILED/tree/halloween-2023) is great.

**Comment by ALEXWARELLC** [comment]
Great, don't remember ever asking it to push the docs part, but if it still works, I'm fine with that.

**Comment by ALEXWARELLC** [comment]
# Notes
## Reviewers
- @louis1706 

## Changes
- Rebased PR so it's up-to-date and shouldn't cause merge conflicts.

## Notes
Everything should be all good for a **Dev CI** run.

**Comment by Misaka-ZeroTwo** [comment]
Duplicate #2171

**Pull request closed without merging**